MAINTENANCE.md

```diff
@@ -163,3 +164,46 @@ the build directory you have to run CMake again manually.
 
 Guides for debugging GMT are provided in the
 [developer resources section](https://docs.generic-mapping-tools.org/dev/debug.html) of the GMT documentation.
+
+## Using build and test aliases
+
+The file [admin/bashrc_for_gmt](/admin/bashrc_for_gmt) contains useful aliases for building and testing GMT that some
+developers chose to use. New pull requests with other aliases that you find helpful are welcome. This file is version controlled, so you should copy the file to a different location in order to edit and use it. For example, use these
+commands to copy it to your home directory:
+
+```
+cd <path-to-gmt>
+cp admin/bashrc_for_gmt ~/bashrc_for_gmt
```

Comment:
Not sure about bashrc best practices, but would it be better to make it a hidden file (with a dot `.`)?

```suggestion
cp admin/bashrc_for_gmt ~/.bashrc_for_gmt
```